bin/hwinj/pycbc_generate_hwinj

```diff
@@ -63,11 +61,11 @@ def _empty_row(obj):
         elif cols[entry] == 'lstring':
             setattr(row,entry,'')
         elif entry == 'process_id':
-            row.process_id = ilwd.ilwdchar("sim_inspiral:process_id:0")
+            row.process_id = 0
         elif entry == 'simulation_id':
-            row.simulation_id = ilwd.ilwdchar("sim_inspiral:simulation_id:0")
+            row.simulation_id = 0
         elif entry == 'event_id':
-            row.event_id = ilwd.ilwdchar("sngl_inspiral:event_id:0")
+            row.event_id = 0
```

Comment:
It would be nice to move this function to the `io/ligolw.py` module (or use the similar one already there) but I will not hold the PR up for this.

Reply:
Agreed, but I wanted to restrict this to "fixing the problem"